### PR TITLE
Added new 2019 devices in the list

### PR DESCRIPTION
**Pull request by wbcyclist**
iPad Air 3, iPad mini 5, Apple TV 4K

### CODE REVIEW

File: DeviceUtil.podspec [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 s.name             = "DeviceUtil"
-s.version          = "5.0.0"
+s.version          = "5.0.1"
 s.summary          = "DeviceUtil helps identifying the exact harware type of the device. e.g. iPhone 6 or iPhone 6s."
 s.homepage         = "https://github.com/InderKumarRathore/DeviceUtil.git"
 s.license          = { :type => "MIT", :file => "LICENSE.txt" }
 s.author           = { "Inder Kumar Rathore" => "contact.mr.rathore@gmail.com" }
   s.social_media_url    = "https://twitter.com/ikr_303"
-s.source           = { :git => "https://github.com/InderKumarRathore/DeviceUtil.git", :tag => s.version }
+s.source           = { :git => "https://github.com/wbcyclist/DeviceUtil.git", :tag => s.version }
```
Comment (armandzwan) [inline DeviceUtil.podspec]: Why this change?

Comment (wbcyclist) [reply DeviceUtil.podspec]: ignore it, i just pull request first commit. the second commit is github default behavior when i push my repo master branch.

Comment (morinokami) [reply DeviceUtil.podspec]: Hi, @wbcyclist. Any progress on this?

I think this line should be reverted back to the original. Github doesn't modify this line by default, so you can fix it and update the pull request.
